### PR TITLE
Added multiple issue templates

### PR DESCRIPTION
**Pull request by pmk21**
Addresses issue #5560

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/BUG_REPORT.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+---
+name: "\U0001F41E Bug Report"
+about: Create a report to help us improve
+title: ''
+labels: Bug
```
Comment (thibaudcolas) [inline .github/ISSUE_TEMPLATE/BUG_REPORT.md]: This should use our existing label for bugs,

```suggestion
labels: 'type:Bug'
```

File: .github/ISSUE_TEMPLATE/FEATURE_REQUEST.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+---
+name: "\U0001F680 Feature Request"
```
Comment (thibaudcolas) [inline .github/ISSUE_TEMPLATE/FEATURE_REQUEST.md]: Same as above,

```suggestion
name: "🚀 Feature request"
```

File: .github/ISSUE_TEMPLATE/FEATURE_REQUEST.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+about: Suggest an idea for improving Wagtail
+title: ''
+labels: Enhancement
```
Comment (thibaudcolas) [inline .github/ISSUE_TEMPLATE/FEATURE_REQUEST.md]:
```suggestion
labels: 'type:Enhancement'
```

**Comment by pmk21** [comment]
Fixed the small issues @thibaudcolas

**Pull request merged**